go.sum

```diff
@@ -623,9 +637,8 @@ github.com/spf13/pflag v1.0.3/go.mod h1:DYY7MBk1bdzusC3SYhjObp+wFpr4gzcvqqNjLnIn
 github.com/spf13/pflag v1.0.5 h1:iy+VFUOCP1a+8yFto/drg2CJ5u0yRoB7fZw3DKv/JXA=
 github.com/spf13/pflag v1.0.5/go.mod h1:McXfInJRrz4CZXVZOBLb0bTZqETkiAhM9Iw0y3An2Bg=
 github.com/spf13/viper v1.3.2/go.mod h1:ZiWeW+zYFKm7srdB9IoDzzZXaJaI5eL9QjNiN/DMA2s=
-github.com/spf13/viper v1.4.0/go.mod h1:PTJ7Z/lr49W6bUbkmS1V3by4uWynFiR9p7+dSq/yZzE=
-github.com/spf13/viper v1.4.1-0.20190911140308-99520c81d86e h1:D4jmJ9BqzeMv7BvAqjooNtXH2PXG7m+pcYRnj2Ojlrk=
-github.com/spf13/viper v1.4.1-0.20190911140308-99520c81d86e/go.mod h1:jUyf+v/KTOnRyUy2/AsjF537WfJWVv3AnlcKSNd+AIg=
+github.com/spf13/viper v1.6.1 h1:VPZzIkznI1YhVMRi6vNFLHSwhnhReBfgTxIPccpfdZk=
+github.com/spf13/viper v1.6.1/go.mod h1:t3iDnF5Jlj76alVNuyFBk5oUMCvsrkbvZK0WQdfDi5k=
```

Comment:
This is the problem. I remember that we had a small problem with viper.

Reply:
oh, grumble. let me see if I can roll just that back. thank you for the institutional wisdom.

Reply:
Time to see if a lower version of golangci-lint will work with go1.14

```
github.com/spf13/viper@v1.4.1-0.20190911140308-99520c81d86e github.com/golangci/golangci-lint@v1.24.0
go get: inconsistent versions:
	github.com/golangci/golangci-lint@v1.24.0 requires github.com/spf13/viper@v1.6.1 (not github.com/spf13/viper@v1.4.1-0.20190911140308-99520c81d86e)
```

Reply:
No, that's not possible. v1.23.6 is the minimum version for go1.14 support, and v1.22.0 is the first version with the new viper dependency.

Reply:
Ok, let me see if I can update viper.

Reply:
It's okay, I manually hacked it for now by setting `override` in bottom of go.mod